dotnet/dynamodb/UpdateItemDataModel/UpdateItemDataModel.cs

```diff
@@ -0,0 +1,170 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved. 
+// SPDX-License-Identifier: MIT-0
+// snippet-start:[dynamodb.dotnet35.UpdateItemDataModel]
+using System;
+// using System.Collections.Generic;
+using System.Configuration;
+// using System.Globalization;
+// using System.Net.Sockets;
+using System.Threading.Tasks;
+
+using Amazon;
+using Amazon.DynamoDBv2;
+using Amazon.DynamoDBv2.DataModel;
+
+namespace DynamoDBCRUD
+{
+    // DO NOT CHANGE THE TABLE ENTRY IN app.config!!!
```

Comment:
Why not? Or what will happen if I do?

Reply:
I've changed the comment to:

// If you change the table name elsewhere
// (as in an app.config in another project),
// you'll have to change it here and rebuild.